src/objective-c/tests/BUILD

```diff
@@ -234,6 +263,23 @@ grpc_objc_ios_unit_test(
     ],
 )
 
+grpc_objc_ios_unit_test(
+    name = "PerfTestsPosix",
+    deps = [
+        "PerfTests-lib-Posix",
+    ],
+)
+
+grpc_objc_ios_unit_test(
+    name = "PerfTests",
+    env = {
+        "GRPC_CFSTREAM_RUN_LOOP": "1",
```

Comment:
AFAIK  GRPC_CFSTREAM_RUN_LOOP shouldn't be necessary anymore when running on the macos monterey images.